blockdevice.go

```diff
@@ -146,9 +167,13 @@ func blockdevice_2_0(source map[string]interface{}) (*blockdevice, error) {
 		"used_size":  schema.ForceUint(),
 		"size":       schema.ForceUint(),
 
+		"filesystem": schema.OneOf(schema.Nil(""), schema.StringMap(schema.Any())),
 		"partitions": schema.List(schema.StringMap(schema.Any())),
 	}
-	checker := schema.FieldMap(fields, nil)
+	defaults := schema.Defaults{
+		"uuid": "",
```

Comment:
Do you even need to specify a default given that above you have:
"uuid":     schema.OneOf(schema.Nil(""), schema.String()),

eg model and id_path are processed the same was as uuid and don't need a default to be set